kubernetes/samples/scripts/create-oracle-db-service/stop-db-service.sh

```diff
@@ -0,0 +1,44 @@
+#!/bin/bash
+# Copyright (c) 2019, Oracle Corporation and/or its affiliates.  All rights reserved.
+# Licensed under the Universal Permissive License v 1.0 as shown at https://oss.oracle.com/licenses/upl.
+
+# Drop the DB Service created by start-db-service.sh
+
+script="${BASH_SOURCE[0]}"
+scriptDir="$( cd "$( dirname "${script}" )" && pwd )"
+source ${scriptDir}/../common/utility.sh
+
+function usage {
+  echo "usage: ${script} -n namespace  [-h]"
+  echo "  -n Kubernate Namespace for Oracle DB Service  (optional)"
+  echo "      (default: default) "
+  echo "  -h Help"
+  exit $1
+}
+
+while getopts ":h:n:" opt; do
+  case $opt in
+    n) namespace="${OPTARG}"
+    ;;
+    h) usage 0
+    ;;
+    *) usage 1
+    ;;
+  esac
+done
+
+
+if [ -z ${namespace} ]; then
+  namespace=default
+fi
+
+
+dbpod=`kubectl get po -n ${namespace}  | grep oracle-db | cut -f1 -d " " `
```

Comment:
This feels brittle...  Why not use jsonpath to pick out the field we want?

Reply:
Some examples: https://kubernetes.io/docs/reference/kubectl/jsonpath/

Reply:
Here we do not have the exact name of the pod, so I need shell utility to get the name of the pod.  If I know the exact pod name I could have used jsonpath. 

$ kubectl get po -n default 
NAME                         READY   STATUS    RESTARTS   AGE
oracle-db-7d569bf5f9-29dt9   1/1     Running   0          78m
Here dbpod returns me the string oracle-db-7d569bf5f9-29dt9